app/components/file/browse/display/file-list-display.html

```diff
@@ -1,37 +1,32 @@
-<bex-quick-list *ngIf="quickList">
-    <!--<bex-quick-list-item *ngFor="let node of nodes" [routerLink]="['/pools', poolId, 'nodes', node.id]" routerLinkActive="selected">-->
-    <!--<bex-quick-list-item *ngFor="let node of nodes" [routerLink]="['/pools', poolId, 'nodes', node.id]" routerLinkActive="selected">
-        <h4 bex-quick-list-item-title>{{node.id}}</h4>
-        <p bex-quick-list-item-field>{{node.state}}</p>
-    </bex-quick-list-item>-->
-    <h1>TODO: Implement quicklist</h1>
-</bex-quick-list>
+<bex-focus-section>
+    <bex-quick-list *ngIf="quickList">
+        <bex-quick-list-item *ngFor="let file of files| fileDirectoryFilter" [routerLink]="['/files', file.url]"
```

Comment:
Font awesome has a few general file icons we could use(text, image, etc.) otherwise maybe some other library